src/index.html

```diff
@@ -7,8 +7,28 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="index.html"><img src="./images/logo.png" alt="Moyo"></a>
```

Comment:
```suggestion
      <a href="index.html">
        <img src="./images/logo.png" alt="Moyo">
      </a>
```
It's better to write a child element from a new line. Otherwise, it's difficult to read.
Also, it's good practice to add an empty line between repetitive sibling elements